### ISSUE TITLE
Slowdown seems off armed police batrider

### ISSUE THREAD

**Issue by theenemy-thd** [psomashekar/Raizing_FPGA#2]
There seems to be more slowdown on the mister than on the arcade.

**Comment by psomashekar** [comment]
I already know about this issue, but cannot address it on MiSTer because
the SDRAM pipeline and bandwidth is too slow. Not to mention, I do not
support MiSTer anymore.

I've already performed a frame by frame assessment against the real board,
so I know why it's slower.

**Comment by theenemy-thd** [comment]
Ahh ok, will you be able to address the issue on the hardware you are now working on?

**Comment by psomashekar** [comment]
Yes. I plan to revamp the code because we have to use it to support dogyuun
and batsugun which is actually more memory intensive from a GP9001
perspective. But, it won't be for MiSTer. It would be for either the pocket
or MARS.

**Comment by theenemy-thd** [comment]
I am glad to hear that the project isnt dead ... I do have a pocket and am enjoying your wrestlefest on it very much.

Very interested to see what Mars is capable of

**Comment by psomashekar** [comment]
It's not dead. It was paused because I was not able to do some things I
wanted to technically in the platform - for example gadgets and then there
are SDRAM bandwidth issues that are contributing to a line glitch on
intense sections.

**Comment by theenemy-thd** [comment]
Look forward to seeing it on Mars, I am gathering its not possible for the analogue pocket to have these changes? Or does it look at memory faster than mister?

Also you mention gadgets above ... what does that mean

*sorry for all the questions, just very interesting to get first hand feedback from the developer of the core. Thanks for all your hard work as well

**Comment by psomashekar** [comment]
No, the pocket memory is better than the mister. I wanted to create an
overlay with gadgets originally so that you get more indicators in-game
about the gameplay status.

I am currently working on midway. I will return back to the raizing video
afterwards.

**Comment by theenemy-thd** [comment]
Really looking forward to seeing these projects, if pocket has any more exclusives I may have to look into that base unit after all, can't wait to see mars in action as well
